include/net/fota_download.rst

```diff
@@ -40,7 +28,19 @@ To use HTTPS instead, apply the changes described in :ref:`the HTTPS section of
 
 The FOTA download library is used in the :ref:`http_application_update_sample` sample.
 
+.. note::
+   To upgrade the upgradable bootloader, you must provide two firmware files: one linked against the S0 partition and one linked against the S1 partition.
+   See :ref:`upgradable_bootloader` for more information.
+
+   To do this, you must provide two paths in the ``file`` argument, separated by a space character:
 
+   *.The first entry must point to the MCUboot variant linked against the S0 partition.
+   * The second entry must point to the MCUboot variant linked against the S1 partition.
+
+   If an upgradeable bootloader is included in the project, the FOTA download library uses the :ref:`lib_secure_services` library to deduce the active MCUboot partition (S0 or S1).
+   If the ``file`` argument contains a space character, it is internally updated to contain only the entry that corresponds to the non-active slot.
+   This updated path is then used to download the image, that MCUboot can copy into the corresponding partition.
+   An additional reset is required for MCUboot to use the upgraded image.
```

Comment:
An additional reset is required for the immutable bootloader to use the upgraded image.

Reply:
Should this be in a note? To me this seems like something we should emphasize in some way.

Reply:
"Note that two resets is required for performing the update of the upgradeable bootloader. The first reset triggers the copy from the update bank to the non-active slot. The second update is needed for the immutable bootloader to select the new and updated version."

Something like that perhaps, explaining why we need the two resets.

Reply:
Agree, more info is definitely better here, to make it possible to understand why it is needed! 

Was also thinking if it might be better so place that in in own section "bootloader update" as the note is getting quite big.

Reply:
Fixed